app/lib/features/chat_ng/utils.dart

```diff
@@ -82,3 +82,12 @@ const List<MaterialAccentColor> chatBubbleDisplayNameColors =
       Colors.orangeAccent,
       Colors.deepOrangeAccent,
     ];
+
+Widget selectableText(BuildContext context, Widget child) => TextSelectionTheme(
```

Comment:
can we move that to `common/toolkit` as a `ActerSelectionArea`-Widget please?!?


generally: helpers that take a `BuildContext` and return a `Widget` and are used more than once, should be a Widget.